civiawards.php

```diff
@@ -242,6 +242,13 @@ function civiawards_addCiviCaseDependentAngularModules(&$dependentModules) {
   $dependentModules[] = "civiawards";
 }
 
+/**
+ * Implements addCiviCaseDependentAngularModules().
+ */
+function civiawards_addWorkflowDependentAngularModules(&$dependentModules) {
+  $dependentModules[] = "civiawards-workflow";
```

Comment:
@deb1990 I understand that there was a new hook added to civicase to support this, but what's the real purpose?

Why not add this dependency directly to civiawards?

Reply:
@reneolivo Right, because we dont want to add all the awards dependency to this page, because its going to bloat this workflows page. Its the same argument for creating the `workflows` module in civicase.

Reply:
Why not create a new angular page using the `civiawards-workflow` angular module and this module includes `civicase-base` and `civiawards` ?

This hook pattern seems to be more difficult to understand than the standard process.

Reply:
This was discussed, and we agreed to keep this as it is.